database/tbcd/database.go

```diff
@@ -397,3 +410,33 @@ func NewTxMapping(txId, blockHash *chainhash.Hash) (txKey TxKey) {
 
 	return txKey
 }
+
+// Helper functions
+
+// B2H converts a raw bloc header to a wire block header structure.
```

Comment:
minor spelling mistake? 😅 
```suggestion
// B2H converts a raw block header to a wire block header structure.
```